### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,35 @@ catkin build -j12 --cmake-args \
...
+# Main workspace with the package
+mkdir -p panda_ws/src
+# Isolated workspace for dependencies
+mkdir -p deps_ws/src
+git clone https://github.com/ArthurH91/panda_torque_mpc.git panda_ws/src/panda_torque_mpc
+vcs import --recursive < panda_ws/src/panda_torque_mpc/panda_torque_mpc.repos deps_ws/src
+cd deps_ws
+source /opt/ros/noetic/setup.bash
+rosdep update --rosdistro $ROS_DISTRO
+rosdep install -y -i --from-paths src --rosdistro $ROS_DISTRO
```
Comment (ArthurH91) [inline README.md:51]: `rosdep install -y -i --from-paths src --rosdistro $ROS_DISTRO  --skip-keys libfranka`

Comment (ahaffemayer) [inline README.md]:
```suggestion
sudo apt update
rosdep update --rosdistro $ROS_DISTRO
rosdep install -y -i --from-paths src --rosdistro $ROS_DISTRO --skip-keys libfranka
```

File: README.md [hunk 2]
```diff
@@ -33,6 +33,35 @@ catkin build -j12 --cmake-args \
...
+> This redusces rebuildigng time, but also increases stability, since
+> [jrl-cmakemodules](https://github.com/jrl-umi3218/jrl-cmakemodules) might cause 
+> ROS path to stop working! To build the package in separate workspace follow:
+```bash
+# Main workspace with the package
+mkdir -p panda_ws/src
+# Isolated workspace for dependencies
+mkdir -p deps_ws/src
+git clone https://github.com/ArthurH91/panda_torque_mpc.git panda_ws/src/panda_torque_mpc
+vcs import --recursive < panda_ws/src/panda_torque_mpc/panda_torque_mpc.repos deps_ws/src
```
Comment (ahaffemayer) [inline README.md]:
```suggestion
mkdir -p ros_ws/src 
# Isolated workspace for dependencies
mkdir -p deps_ws/src
git clone https://github.com/ArthurH91/panda_torque_mpc.git ros_ws/src/panda_torque_mpc
vcs import --recursive < ros_ws/src/panda_torque_mpc/panda_torque_mpc.repos deps_ws/src
```

File: README.md [hunk 3]
```diff
@@ -33,6 +33,35 @@ catkin build -j12 --cmake-args \
...
+source /opt/ros/noetic/setup.bash
+rosdep update --rosdistro $ROS_DISTRO
+rosdep install -y -i --from-paths src --rosdistro $ROS_DISTRO
+catkin build -j12 --cmake-args \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_UNIT_TEST=OFF \
+    -DBUILD_PYTHON_INTERFACE=OFF \
+    -DBUILD_WITH_URDF_SUPPORT=ON \
+    -DBUILD_WITH_COLLISION_SUPPORT=ON
+cd ../panda_ws
```
Comment (ahaffemayer) [inline README.md]:
```suggestion
cd ../ros_ws
```

**Review by ahaffemayer — CHANGES_REQUESTED** [review]
that's nice but maybe switch from `panda_ws` to `ros_ws` to match the container's name

**Review by ahaffemayer — COMMENTED** [review]
lgtm

**Review by ahaffemayer — APPROVED** [review]

**Pull request merged**